src/main/java/com/tabnine/statusBar/TabnineStatusBarWidget.java

```diff
@@ -108,6 +116,9 @@ private ServiceLevel getServiceLevel() {
   // Compatability implementation. DO NOT ADD @Override.
   @Nullable
   public String getTooltipText() {
+    if (Config.IS_ON_PREM) {
```

Comment:
when clicking should open the tabnine settings - see https://github.com/codota/tabnine-intellij-enterprise/blob/master/src/main/java/com/tabnine/statusBar/TabnineStatusBarWidget.java#L67

Reply:
this doesn't work :(